Dockerfile

```diff
@@ -58,7 +58,13 @@ RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 COPY requirements.txt /usr/src/app/
-RUN pip install -r requirements.txt
+RUN set -x \
+    && deps='gcc librdkafka-dev libc-dev' \
```

Comment:
Why do we need this ?

Reply:
confluent-kafka doesn't have a prebuilt on arm64 sadly